ingestion/src/metadata/utils/secrets_manager.py

```diff
@@ -36,24 +46,40 @@
     "ServiceConnectionType", ServiceConnection.__fields__["__root__"].type_
 )
 
+AuthProviderClientType = NewType(
+    "AuthProviderClientType", OpenMetadataConnection.__fields__["securityConfig"].type_
```

Comment:
hehe

Reply:
I know I did not do it on the topology, but we might want to add a comment explaining what this is about